index.js

```diff
@@ -59,7 +66,8 @@ const TAX_CREDIT_VALUE = {
 	2019:218,
 	2020:219,
 	2021:218,
-	2022:235}; // NIS per month
+	2022:235,
```

Comment:
Tax credit value for 2022 should be 223 not 235. I realise this mistake has been in there for a while. Please update since you're already editing the file.